syntax = "proto3";

package orderqueue;

service OrderQueueService {
    rpc EnqueueOrder (EnqueueRequest) returns (EnqueueResponse);
}

message Order {
    string orderId = 1;
    string userName = 2;
}

message EnqueueRequest {
    Order order = 1;
}

message EnqueueResponse {
    bool success = 1;
    string message = 2;
}